### DISCUSSION TITLE
Mastodon Backups

### DISCUSSION THREAD

**Discussion by hairylarry** [mastodon/mastodon#30364] [Q&A] [answered]
In /mastodon/live/public/system is it necessary to backup the cache directory? Are these only files from other servers? Are the user uploaded files kept in the other directories?

**Comment by vmstan** [comment, marked as answer]
My backup of this data excludes the `cache` directory but with the understanding if something happened where I needed to restore that data, I would have to do some work to force Mastodon to repopulate it.

**Reply by hairylarry** [reply]
That's what I thought. I'm more concerned about saving the files uploaded to my server than backing up other people's servers for convenience. The cache directory is large and hard to backup even when minimizing it in the settings.

So I'm going with 

everything in live/public/system except the cache directory
pg_dump for postgresql
the .env.production file
/var/lib/redis/dump.rdb for redis

Am I missing anything?

Thanks
